fractal_client/cmd/_workflow.py

```diff
@@ -126,28 +131,14 @@ def patch_workflowtask(
     project_id: int,
     workflow_id: int,
     workflow_task_id: int,
-    args_file: Optional[str] = None,
-    meta_file: Optional[str] = None,
-) -> RichJsonInterface:
-
-    # Check that at least one of args_file or meta_file was given (note: it
-    # would be reasonable to check it in the parser, but we are not aware of a
-    # method within argparse).
-    if not (args_file or meta_file):
-        raise ValueError(
-            "At least one of {args_file,meta_file} arguments is required"
-        )
-
-    # Combine args/meta payload
+    input_filters: str,
```

Comment:
Include all five attributes of https://github.com/fractal-analytics-platform/fractal-server/blob/main/fractal_server/app/schemas/v2/workflowtask.py#L78